engine/dlib/src/dlib/android.cpp

```diff
@@ -50,11 +50,11 @@ bool ThreadAttacher::Detach()
     return ok;
 }
 
-jclass LoadClass(JNIEnv* env, const char* class_name)
+jclass LoadClass(JNIEnv* env, jobject activity, const char* class_name)
```

Comment:
It's important to have such method to be able to use it in `OnCreate` hook we provide in SDK